utils.go

```diff
@@ -131,6 +131,7 @@ func updatePluginDependencies(gitURL string) (err error) {
 func goGet(gitURL string) (err error) {
 	args := []string{"get", "-v", "-d", "-buildmode=plugin", gitURL}
 	goget := exec.Command("go", args...)
+	goget.Env = append(os.Environ(), "GO111MODULE=off")
```

Comment:
@djeebus @itsmontoya should we also support reading `GOPRIVATE=...` from config here?

Reply:
Would reduce the number of steps necessary to configure env in onramping

Reply:
Also.. should we analogously set `GO111MODULE=on` when we do `go build` in the other portion?

Reply:
This env is only passed for the go get command. Should be good :)

Reply:
@dhalman probably out of scope for this PR. Can you get a PR up with a PoC?